addons/website/views/snippets.xml

```diff
@@ -353,6 +353,47 @@
     </section>
 </template>
 
+<template id="s_countdown" name="Countdown">
+    <section class="s_countdown"
+        data-display="dhms"
+        data-end-action="nothing"
+        data-layout="light"
+        data-size="175"
+        data-end-time="1604140819">
+        <div class="container">
+            <div class="row">
+                <div class="col text-center mt8 mb8">
```

Comment:
In fact, should not we make this snippet an inner content snippet without section and container etc ? cc @sbuhl

Reply:
Was this discussed finally?